src/comp/fieldView.js

```diff
@@ -11,9 +11,9 @@ export default function FieldView(props) {
         canvas.width = 400;
         canvas.height = 400;
         const ctx = canvas.getContext("2d");
-        const newField = new Field(ctx, props.baseColor, props.size);
-        newField.renderAll();
-        newField.progressMawer();
+        const newField = new Field(ctx, props.baseColor, props.grownColor, props.mawerColor, props.size);
+        newField.initiate();
+        newField.mawer.progress(props.size);
```

Comment:
czemu `Mawer.progress` przyjmuje argument? tak zawsze było?

Reply:
Tak, to się wiąże ze stworzeniem listy obiektów Field, czego nie chciałem robić na tym branczu. Stwierdziłem że za dużo bym tutaj zrobił.